client/src/dummyData/dropDownItems.js

```diff
@@ -16,7 +16,20 @@ const locations = [
     { id: 223, name: 'Florida' }
 ]
 
+const messages = [
+    {
+        content: "blah blah"
+    },
+    {
+        content: "etc etc"
+    },
+    {
+        content: "yada yada"
+    }
+]
+
```

Comment:
I know this is placeholder data, was this ticket to hook it up or not? looks ok anyhow

Reply:
No the hookup was not part of the request. And I was under the impression that the alerts would come from websockets -- is that your understanding as well?

Reply:
We can probably handle hooking up message notifications in a later ticket

Reply:
yeah -- messages were to be hooked up afterwards. Let's talk about how to go about that.